javascript/organizing_your_javascript_code/project_restaurant_page.md

```diff
@@ -53,6 +49,4 @@ Let's use what we've learned and take a chance to continue practicing DOM manipu
             - Now each  time you need to update your project's live preview, you `npm run <YourScriptName>` in your project's terminal.
             - To learn more about this, here's a short tutorial video on [Node Package Manager Scripts](https://www.youtube.com/watch?v=REdzp64dijs).
 
-    1. Recall that the **source branch** for GitHub Pages is set in your repository's settings.
-
-</div>
```

Comment:
Could you add the `</div>` back on the last line please then add a single blank line after it?

Reply:
Done